local-storage/operator/overlays/stable/kustomization.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: openshift-local-storage
+
+bases:
+  - ../../base
+
+patchesJson6902:
```

Comment:
Use `patches` instead of the `patchesJson6902`.

We still have patchesJson6902 in the repo but it is on my todo list to clean some of these up.

In case you are curious of the difference, I like this description here:
https://stackoverflow.com/questions/63604579/what-is-the-difference-between-patches-vs-patchesjson6902-in-kustomize